### ISSUE TITLE
Homebrew support

### ISSUE THREAD

**Issue by thesabbir** [jwise/HoRNDIS#39]
It would be nice if we can install HoRNDIS with [homebrew](http://brew.sh/)

**Comment by jwise** [comment]
I think you can?

**Comment by thesabbir** [comment]
Never done it before. Just a heavy brew user :)

**Comment by jwise** [comment]
I am pretty sure there is a brew package already; that said, I'm pretty sure brew can't install kexts on its own, too... 

Closing; feel free to reopen if package does not exist

**Comment by SimenB** [comment]
Cask seems to have it, untested though (I'll test it this weekend, can report back then)

https://github.com/caskroom/homebrew-cask/blob/master/Casks/horndis.rb

**Comment by joelpurra** [comment]
`brew cask install horndis` *and a reboot* worked very well for me!

Thank you very much @jwise -- now I don't have to enable/disable the buggy Bluetooth PAN to reconnect all the time =)

**Comment by binaricko** [comment]
Can confirm that `brew cask install horndis` plus reboot worked for me on OSX Sierra.  Oddly, the `pkg` installer didn't work ... even though it looks like the brew installer downloads the same file?  But I'd had previous versions of HoRNDIS left over from before the Sierra upgrade, which I cleaned out manually before trying the Homebrew install.

tl;dr: After an upgrade to Sierra:
```bash
sudo rm -R /Library/Extensions/HoRNDIS.kext
sudo rm -R /System/Library/Extensions/HoRNDIS.kext
brew cask install horndis
shutdown -r now
```
